admin_guide/encrypting_data.adoc

```diff
@@ -0,0 +1,264 @@
+[[admin-guide-encrypting-data-rest]]
+= Encrypting Data at Rest
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
```

Comment:
I don't know whether I need `:experimental:` and `{product-author}`, `{product-version}`. I've copied them over from another topic but I don't know what they are doing.